js/.jscsrc

```diff
@@ -1,4 +1,5 @@
 {
+    "requireSemicolons": true,
```

Comment:
The issue with standard js is that it's underspecified, just a set of loose "don't do this" rules that don't create a full style guide. I'm not sure I understand the advantage of the project over a standardized (and more specific) eslint config. However, I can buy their semicolon argument; they are essentially redundant in javascript.